torch_xla/experimental/custom_kernel.py

```diff
@@ -968,23 +969,6 @@ def ragged_paged_attention(
     num_queries_per_block=None,
     vmem_limit_bytes=None,
 ):
-  if mask_value is None:
```

Comment:
why remove this part?

Reply:
Thanks for catching this. It's removed by accident.